numba/targets/codegen.py

```diff
@@ -227,10 +229,13 @@ class BaseCPUCodegen(object):
     def __init__(self, module_name):
         self._libraries = set()
         self._data_layout = None
-        self._llvm_module = ll.parse_assembly(
-            str(self._create_empty_module(module_name)))
+        self._llvm_module = self._materialize_module(
+            self._create_empty_module(module_name))
         self._init(self._llvm_module)
 
+    def _materialize_module(self, ir_module):
```

Comment:
Maybe just a nitpick, but this is used outside the `BaseCPUCodegen` class - should it be `materialize_module` (no underscore)?